### PR TITLE
Add snyk ci

### PR DESCRIPTION
**Pull request by Param-29**
## Description
Running Snyk currently with PRs and my private key. 
- [x] Created new file to run after code is merged. 

To do: 
- [ ] add `secrets.SNYK_TOKEN` variable
- [ ] Remove private key and synk from PR_tests.  
## Affected Dependencies
None

## How has this been tested?
Testing on CI
## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by gmuraru** [comment]
Do you know if this would also benefit SyMPC? :D - thinking of opening an issue there

**Comment by Param-29** [comment]
Yeah, these were suggested by the security team 
Feature request details can be found here: https://github.com/OpenMined/PySyft/issues/5448 
I'd love to integrate these in SyMPC repos as well if you'd like that xD

**Comment by gmuraru** [comment]
Sure!

If you have time and you know the workflow I think that would be great! You can open an issue and send it to me -- I will assign you to it.

File: .github/workflows/security.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      - name: Install supported library
+        run: |
+          pip install -r requirements.libs.deps.txt --default-timeout=60
+          pip list
+
+      - name: Synk security check
+        if: startsWith(runner.os, 'ubuntu')
+        uses: snyk/actions/python-3.8@master
+        env:
+          SNYK_TOKEN: 3106d1f2-a045-4bc6-8695-d899ab653448 # to be changed to ${{ secrets.SNYK_TOKEN }}
```
Comment (gmuraru) [inline .github/workflows/security.yml:77]: Q: @madhavajay or @tudorcebere I think you need to create a token for this?

**Comment by Param-29** [comment]
Closing this, continued here https://github.com/OpenMined/PySyft/pull/5595

**Pull request closed without merging**